src/main/resources/public/html/realm.templates.html

```diff
@@ -98,7 +98,7 @@ <h3>Templates</h3>
                             <ul class="link-list">
                                 <li>
                                     <a class="list-item left-icon"
-                                        ui-sref="realm.template({realmId:item.realm, templateId:template.id})">
+                                        ui-sref="realm.template({realmId:item.realm, templateId:item.id})">
```

Comment:
wasn't this already fixed in the other pull request? if so, drop from here and rebase